blob/fileblob/fileblob.go

```diff
@@ -88,6 +88,9 @@ type URLOpener struct{}
 //  - file://localhost/c:/foo/bar
 //    -> Also passes "c:\foo\bar".
 func (*URLOpener) OpenBucketURL(ctx context.Context, u *url.URL) (*blob.Bucket, error) {
+	for param := range u.Query() {
```

Comment:
Should unsupported query parameters just be ignored? I think that's more common than error out.

Reply:
This is consistent with how it's handled elsewhere. My first thought is that it's better to error out than just silently ignore things, but I haven't thought about it that much. If you disagree, can you file a separate issue to discuss?

Reply:
Filed #1435